src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -376,13 +476,19 @@
     
     /**
      * When true, realtime updates are ignored during this search.
+     *
+     * TODO Needs to be implemented
      */
+    @Deprecated
     public boolean ignoreRealtimeUpdates = false;
 
     /**
      * If true, the remaining weight heuristic is disabled. Currently only implemented for the long
      * distance path service.
+     *
+     * @Deprecated - Not needed in Raptor
      */
+    @Deprecated
     public boolean disableRemainingWeightHeuristic = false;
```

Comment:
INFO: Heuristic (and reverse search) is part of "profiles" in the Raptor implementation. Profiles are carefully put together to get a working set of parameters that work together. Choosing a profile should be possible from the API, but not setting specific parameters to enable strategies.